code/index.html

```diff
@@ -42,6 +55,7 @@ <h1>Does the person have</h1>
       </select>
 
       <button id="filter" class="filled-button">FIND OUT</button>
+      <div class="guess-counter">Number of Guesses: <span id = 'guesses'>0</span></div>
     </aside>
 
     <section class="board-wrapper">
```

Comment:
This "board-wrapper"  is still showing on your win and lose screens.

Reply:
I will look into this further; see comment below